Workbooks/KeyVault/AzureMonitor/AzureMonitor.workbook

```diff
@@ -472,6 +472,7 @@
                     "formatter": 13,
                     "formatOptions": {
                       "linkTarget": "Resource",
+                      "subTarget": "insights",
```

Comment:
for now keyvault resources don't have insights right? we will add this later?

Reply:
I think they do now? Or rather once their PR is deployed they will.